Source/iOS/OIDExternalUserAgentIOS.m

```diff
@@ -61,82 +61,93 @@ - (BOOL)presentExternalUserAgentRequest:(id<OIDExternalUserAgentRequest>)request
 
   _externalUserAgentFlowInProgress = YES;
   _session = session;
-  BOOL openedSafari = NO;
+  BOOL openedUserAgent = NO;
   NSURL *requestURL = [request externalUserAgentRequestURL];
 
   // iOS 12 and later, use ASWebAuthenticationSession
+  // ASWebAuthenticationSession doesn't work with guided access (rdar://40809553)
```

Comment:
Can you move this comment to just above the if-statement for `UIAccessibilityIsGuidedAccessEnabled`? And below

Reply:
Sure.

Reply:
Sorry I just meant just the UIAccessibilityIsGuidedAccessEnabled comment, not both. I think the comment should precede the if-statement that it relates to.

Reply:
Done! =)